### PR TITLE
[PROJECT ONBOARDING] argocd-vault-plugin

### PR DESCRIPTION
**Pull request by werne2j**

### CODE REVIEW

**Review by sbose78 — APPROVED** [review]

File: proposals/argocd-vault-plugin.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Once you are ready, please help the reviewer understand your project better by
+answering the following questions in your onboarding proposal:
+
+- [x] What is your project repository Github URL? - **https://github.com/IBM/argocd-vault-plugin**
+- [x] Do you wish to host your project repository https://github.com/argoproj-labs or https://github.com/argoproj? - **https://github.com/argoproj-labs**
+- [x] Does your project focus on enhancing or providing additional features to one of the existing [core
+ projects](https://github.com/argoproj/argoproj#what-is-argoproj) ? If yes, which of the core projects is your proposed project related to? - **https://github.com/argoproj/argo-cd**
+- [x] Is it endorsed by any of the Argo subproject maintainers? Please mention sponsors from the subproject. - **@sbose78**
+- [x] How does it align with the goals of the Argo community? - **argocd-vault-plugin adds an unofficial secret management plugin that can be used in tandem with Argo CD or could be used independently as part of an Argo Workflow.**
```
Comment (sbose78) [inline proposals/argocd-vault-plugin.md:13]: Could you please share a detailed example of how this could be used with Argo CD ?

**Review by alexmt — APPROVED** [review]
+1. Thank you

**Comment by jannfis** [comment]
+1 awesome stuff

**Comment by jkayani** [comment]
@alexmt @sbose78 @jannfis Hey guys - just checking to see if anything else needed on our end to get this merged. Happy to update the commit message to be semantic if needed.

**Comment by sbose78** [comment]
I will go ahead and create the repo https://github.com/argoproj-labs/argocd-vault-plugin

**Comment by alexec** [comment]
Can this be merged? Or closed?

**Comment by edlee2121** [comment]
Any further comments?

**Comment by werne2j** [comment]
As far as I am concerned this can be closed. Not sure if the goal is to merge this in or just close it as is but I think we (me and @jkayani are good on our side)

**Pull request merged**